default.json

```diff
@@ -56,8 +72,13 @@
         "mcr.microsoft.com/dotnet/aspnet",
         "mcr.microsoft.com/dotnet/runtime",
         "mcr.microsoft.com/dotnet/runtime-deps",
-        "/^[mM]icrosoft\\./",
-        "/^[sS]ystem\\./"
+        "Azure.Identity",
+        "/^Microsoft\\./i",

```

Comment:
It's simpler to use the i flag for case insensitive:
https://docs.renovatebot.com/string-pattern-matching/#regex-is-case-sensitive-by-default